### PR TITLE
Additional material to course content

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Now I send pull request to `nesterone:master` successfully. Thanx a lot for your advice, [@nesterone](https://github.com/nesterone)!

### CODE REVIEW

**Comment by artem-konovalov-personal** [comment]
And if I understand well, now I should send changes into `nesterone:master` only from my master-branch `nervebassmaster:master`. Am I right?
P.S. I have closed the previous wrong pull request

**Comment by artem-konovalov-personal** [comment]
I have added correct link to the **Modules** topic from **Eloquent JavaScript**

File: 01-syntax/02-functions-variables-scope-and-memory/min.js [hunk 1]
```diff
@@ -10,3 +10,5 @@ console.log(min(0, 10));
 // → 0
 console.log(min(0, -10));
 // → -10
+console.log(min(0, -10));
```
Comment (nesterone) [inline 01-syntax/02-functions-variables-scope-and-memory/min.js]: ;-) 

More test cases, they looks quite similar

File: 01-syntax/03-data-structures/the-sum-of-ranges/range.js [hunk 2]
```diff
@@ -9,3 +9,7 @@
 console.log(range(5, 2, -1));
 
 // → [5, 4, 3, 2]
+
+console.log(range(10, 1));
```
Comment (nesterone) [inline 01-syntax/03-data-structures/the-sum-of-ranges/range.js:13]: More test cases... 
For example
What about calling range with only one parameter ?

**Comment by nervebassmaster** [comment]
Closed [#408](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/408)

**Comment by artem-konovalov-personal** [comment]
Closed [#408](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/408)

**Comment by artem-konovalov-personal** [comment]
Also I have added five additional test cases to the **deepComparison** topic for the course content

**Pull request merged**